test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1

```diff
@@ -133,6 +133,27 @@ $testCases = @{
 $modulesInBox = @("Microsoft.PowerShell.Core"
                   Get-Module -ListAvailable | ForEach-Object{$_.Name}
 )
+function PathIsReadOnly
```

Comment:
Can we move this function to a common file and then dot source it in both the test files.